lib/src/main/java/graphql/nadel/NextgenEngine.kt

```diff
@@ -541,4 +544,27 @@ internal class NextgenEngine(
             field,
         )
     }
+
+    private fun makeTimer(
+        operationDefinition: OperationDefinition,
+        executionInput: ExecutionInput,
+        latencyTracker: NadelInternalLatencyTracker,
+        instrumentationState: InstrumentationState?,
+    ): NadelInstrumentationTimer {
+        val isTimerEnabled = instrumentation.isTimingEnabled(
+            params = NadelInstrumentationIsTimingEnabledParameters(
+                instrumentationState = instrumentationState,
+                context = executionInput.context,
+                operationName = operationDefinition.name,
```

Comment:
Now passes more info to the `isTimingEnabled` function. Enough info to evaluate a FF per GraphQL operation